docs/faq.md

```diff
@@ -4,7 +4,7 @@ LocalPV-ZFS is a CSI driver for dynamically provisioning a volume in ZFS storage
 
 ### 2. How to install LocalPV-ZFS
 
-Make sure that all the nodes have zfsutils-linux installed. We should go to the each node of the cluster and install zfs utils
+Make sure that all the nodes have zfsutils-linux installed. We should go to the each node of the cluster and install zfs utils.
```

Comment:
Do we need to specify any version of zfsutils-linux here?

Reply:
Also need to ensure they're available on a specific path location right? Otherwise need to override it on the chart?

Reply:
> Do we need to specify any version of zfsutils-linux here?

That is specified in the README.

Reply:
> Also need to ensure they're available on a specific path location right? Otherwise need to override it on the chart?

No we check at all the possible places and use wherever we find them. So mostly should be fine.

Reply:
> we check at all the possible places

No we don't, this doesn't work on `NixOS` by default. And probably won't on other similar distros.
You have to either use custom location or create bind mounts for this to work.

Reply:
Done.